doc/md/fundamentals/1-basic-syntax/8-functions.md

```diff
@@ -9,31 +9,31 @@ Functions in Motoko can take on a variety of different attributes, the most basi
 
 The most basic Motoko [function declaration](https://internetcomputer.org/docs/motoko/fundamentals/declarations/function-declarations) is:
 
-```motoko
-func exampleFunction() {};
+```motoko no-repl
+func exampleFunction() : () {};
 ```
 
 By default, all functions are declared private unless explicitly declared `public`.
 
 ```motoko no-repl
-public func exampleFunction() {};
+public func exampleFunction() : () {};
```

Comment:
````suggestion
object Counter  {
   var value = 0;
   func reset() { value := 0 };
   public func inc() { value := 1};
   public func get() : Nat { value }; 
}
```
The object `Counter` has two public methods, the functions `Counter.inc()` and `Counter.get()`. Both `value` and `reset()` are implicitly `private` - attempts to access `Counter.reset()` and `Counter.value` are type errors.
````
But maybe it would be best to discuss `public` and `private` in `6-objects-classes.md`.